app/guides/page.tsx

```diff
@@ -0,0 +1,9 @@
+"use server";
+
+export default async function GuidesPage() {
+  return (
+    <>
```

Comment:
Don't need the Fragment if only returning a single element 👍

Reply:
Ignore this for now, it will be changing very soon when we get read if the below and add more content..